ocs_ci/utility/version.py

```
@@ -150,25 +151,62 @@ def get_ocp_version(seperator=None):
+            f"HTTP error {r.status_code} when retrieving GA version for channel {channel}: {http_err}"
+        )
+        raise
+    except requests.exceptions.Timeout as timeout_err:
+        log.error(
+            f"Timeout when retrieving GA version for channel {channel}: {timeout_err}"
+        )
+        raise
+    except requests.exceptions.RequestException as req_err:
+        log.error(
+            f"Request error when retrieving GA version for channel {channel}: {req_err}"
+        )
```

Ruff (0.15.20)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)